episodes/18-preparing.md

```diff
@@ -48,27 +47,57 @@ but reduces a lot the preparation time of future lessons and are super useful to
 
 \- Dr. Allegra Via, Carpentries Instructor and Instructor Trainer
 
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+::::::::::::::::::::::::::::::::::::::  challenge
+
+## Exercise: Prepare a teaching plan (15 minutes)
+
+Create a bullet point list or brief notes describing
+what you will say and do when teaching the section you have been focussing on
+during this training.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-- Using figures and visual aids
-  - Remember the alt text
-  - Call back to placing explanatory text adjacent to relevant part of visualisation
-- Working with the lesson infrastructure:
-  - Setup instructions
-    - This is time well-invested. Clear instructions now may give you more time when teaching the lesson.
-  - Discuss briefly what belongs in Instructor Notes pages:
-    - strengths and weaknesses of current design
-    - what has been tried before and what worked/did not work
-    - tips for teaching the lesson/particular sections
-    - challenges encountered
-    - parts that seem to cause biggest trouble/most confusion for learners (i.e. where to tread carefully), etc
-- Collecting feedback
-  - a reminder of some of the methods for collecting feedback that can be most helpful for a trial run
-    - minute cards
-    - one up, one down
-    - perhaps even some more specific feedback, for a relatively short trial run e.g. how clear were the exercises? were they appropriately challenging i.e. learner felt like they had to think about it to complete the challenge, but not so difficult that they gave up/became frustrated.
-  - link to relevant section of Instructor Training
+Figures and other visual aids are great tools to help participants understand the content being presented and make associations between pieces of information. If you are planning on using these in your lesson or teaching session - remember to enhance them with the ALT text (invisible description of images which are read aloud to blind users on screen readers providing alternative text-based format for images) or place explanatory text adjacent to relevant part of visualisation.
+  
+If your lesson requires participants to use any software or tools - it is worthwhile investing time now into creating clear setup and installation instructions for participants to follow and circulating them ahead of the workshop. This will give you more time at the pilot for teaching the lesson as troubleshooting installation problems on the spot is often time-consuming and stressfull. Each Carpentries lesson has a dedicated place for this purpose within The Carpentries Workbench located in the file `setup.md` in the lesson project root. 
```

Comment:
```suggestion
If your lesson requires participants to use any software or tools - it is worthwhile investing time now into creating clear setup and installation instructions for participants to follow and circulating them ahead of the workshop. This will give you more time at the pilot for teaching the lesson as troubleshooting installation problems on the spot is often time-consuming and stressful. Each Carpentries lesson has a dedicated place for setup instructions within The Carpentries Workbench located in the file `setup.md` in the lesson project root.
You may also want to include directions to download any data needed in the setup.
```

Reply:
Sorry several suggestions are all linked in this comment.  Wasn't sure how to separate them without more line breaks.